strawberryfields/ops.py

```diff
@@ -940,17 +941,16 @@ class Dgate(Gate):
     or use the polar form :math:`a = r, \phi` and still get the same result.
 
     Args:
-        a (complex): displacement parameter :math:`\alpha`
-        phi (float): extra (optional) phase angle :math:`\phi`
+        r (float): displacement amplitude :math:`|\alpha|`
+        phi (float): displacement angle :math:`\phi`
     """
 
-    def __init__(self, a, phi=0.0):
-        super().__init__([a, phi])
+    def __init__(self, r, phi=0.0):
```

Comment:
@ziofil readded the default value of `phi` here, but let me know if you'd rather it was removed!

Reply:
One thing we chatted about would be removing the default value again and adding a warning here. This would help users get accustomed to the new signature and keep in mind what arguments need to be passed